src/languages/en.js

```diff
@@ -970,4 +970,18 @@ export default {
         },
         refresh: 'Refresh',
     },
+    fileDownload: {
+        success: {
+            title: 'Downloaded!',
+            message: 'Attachment successfully downloaded',
+        },
+        generalError: {
+            title: 'Attachment Error',
+            message: 'Attachment cannot be downloaded',
+        },
+        permissionError: {
+            title: 'Access Needed',
+            message: 'NewExpensify does not have access to save attachments. To enable access, tap Settings and allow access.',
```

Comment:
I think we'll want to have this copy updated.  Going to check with the team and get back to you on this.  This also doesn't seem to be clear enough for both platforms

Reply:
Sure